roles/kolibri/files/kolibri.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Kolibri server
+
+[Service]
+User=ideascube
+Group=ideascube
+ExecStart=/home/ideascube/.kolibri/env/bin/python2 /home/ideascube/.kolibri/env/bin/kolibri start
```

Comment:
From https://libraries.io/pypi/kolibri : both are acceptable. Do we have a preference for Python3?